composer.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_readme": [
         "This file locks the dependencies of your project to a known state",
-        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
```

Comment:
Update your Composer to avoid this change.

Reply:
(Wonder if we should do something in CI to force the latest format? Maybe a `composer update --lock` and see if there's changes?)

Reply:
The way we are trying to solve the same problem with `pip` is to use the `pip`|`composer` in the container, which should be identical everywhere.

Reply:
We shall have a `dependencies` pipeline that can update it out of band.